### DISCUSSION TITLE
Nightcore/Daycore with adjusted speed pitch fix

### DISCUSSION THREAD

**Discussion by ChenMerami** [ppy/osu#22354] [Ideas] [answered]
Can you please fix the pitch of the song with NC/DC adjusted speed applied? It is done right in the editor, so i dont think it is impossible. For example check the editor 50% speed and daycore 50% speed. Daycore one is distorted so much while the editor sound good. NC: below x1.5 it is distorted and after x1.5 it is overdone and with DC: below x0.75 it is distorted and higher x0.75 it is overdone (you can clearly hear it with x0.99 speed).
I really want to practice stream maps in lazer with daycore around 0.9 speed but it just sounds so bad. Same when i try to push my limits with NC 1.1-1.25, it is just so distorted.

**Comment by mk56-spn** [comment, marked as answer]
https://github.com/ppy/osu/discussions/13125 please search before asking things .
